docs/modules/links.md

```diff
@@ -0,0 +1,125 @@
+# Dynamic Links
+
+[Firebase Dynamic Links](https://firebase.google.com/docs/dynamic-links/) allows you to create and receive links on both Android and iOS platforms. Assuming the installation instructions have been followed, Firebase Dynamic Links is ready to go.
+
+
+All Dynamic Links operations are accessed via `firebase.links()`
+
+## Create Dynamic Links
+
+RNFirebase mimics [The REST API](https://firebase.google.com/docs/dynamic-links/rest) for Dynamic Links creation.
+The differences from the REST API are:
+1. The input for the methods is a javascript object instead of a JSON object.
+2. The response contains the URL string only.
+
+### Methods
+
+#### `createDynamicLink(parameters: Object): Promise<String>`
+
+Creates a long dynamic link.
+
+```javascript
+firebase.links().createDynamicLink({
+  dynamicLinkInfo: {
+    dynamicLinkDomain: "abc123.app.goo.gl",
+    link: "https://example.com/",
```

Comment:
I would change `link: "https://example.com/",` to `link: "https://example.com?param1=foo&param2=bar",` so it is clear you can specify parameters